src/blockly/generators/propc/gpio.js

```diff
@@ -976,19 +968,18 @@ Blockly.Blocks.fb360_set = {
                 .appendField(new Blockly.FieldDropdown(this._menuOptions, function (param) {
                     this.sourceBlock_.updateShape_(param);
                 }), "PARAM")
-                .appendField('R,-720,720,0', 'RANGEVALS1');
-        this.getField('RANGEVALS1').setVisible(false);
+                .appendRange('R,-720,720,0', 'RANGEVALS1');
```

Comment:
This call appears to be inconsistent with similar calls in this file. Specifically, the third parameter - 'RANGEVALS1' - is omitted in the other calls to appendRange().

Reply:
Fixed in latest commits